packages/expo-notifications/src/getExpoPushTokenAsync.ts

```diff
@@ -20,20 +20,44 @@ interface Options {
   type?: string;
   deviceId?: string;
   development?: boolean;
-  experienceId?: string;
-  projectId?: string;
+  projectId: string;
   applicationId?: string;
   devicePushToken?: DevicePushToken;
 }
 
-export default async function getExpoPushTokenAsync(options: Options = {}): Promise<ExpoPushToken> {
+interface DeprecatedOptions extends Omit<Options, 'projectId'> {
```

Comment:
```suggestion
interface DeprecatedOptions extends Omit<Options, 'projectId'> {
```

We prefer using `type` instead of `interface` for representing types.

Reply:
Also, can we deprecate the filed in the current options instead of introducing new type to reduce the types complexity?

Reply:
Sure, I've updated the PR based on @wschurman comments and things are much simpler now